server/http_handler_test.go

```diff
@@ -584,6 +584,37 @@ func (ts *HTTPHandlerTestSuite) TestGetSchema(c *C) {
 	se, err := session.CreateSession(ts.store.(kv.Storage))
 	c.Assert(err, IsNil)
 	c.Assert(dbtbl.SchemaVersion, Equals, domain.GetDomain(se.(sessionctx.Context)).InfoSchema().SchemaMetaVersion())
+
+	db, err := sql.Open("mysql", getDSN())
+	c.Assert(err, IsNil, Commentf("Error connecting"))
+	defer db.Close()
+	dbt := &DBTest{c, db}
+
+	dbt.mustExec("create database if not exists test;")
+	dbt.mustExec("use test;")
+	dbt.mustExec("set @@tidb_enable_table_partition = 1")
```

Comment:
why we need to set `tidb_enable_table_partition`(compared with https://github.com/pingcap/tidb/pull/10026)?

Reply:
Because release2.1 haven't enable create partition table default, But master default is enable.

Reply:
Got it, thanks